Lib/test/test_fstring.py

```diff
@@ -338,7 +338,7 @@ def g():
         self.assertIsNone(g.__doc__)
 
     def test_literal_eval(self):
-        with self.assertRaisesRegex(ValueError, 'malformed node or string'):
+        with self.assertRaisesRegex(ValueError, 'JoinedStr not allowed in literal'):
```

Comment:
Is it possible to say instead: `f-string not allowed in literal`?

Reply:
I think this is pretty simple and straight forward, no need for that. That operation requires a big static dict that maps every node with description like `BinOp: arithmetic operation JoinedStr: f-string FunctionDef: Function definition`...

Reply:
I don’t think the message is straightforward for readers, who mostly won’t know what a JoinedStr is :slightly_smiling_face: